module/documents/item.mjs

```diff
@@ -1609,6 +1609,16 @@ export default class Item5e extends SystemDocumentMixin(Item) {
   async rollDamage({critical, event=null, spellLevel=null, versatile=false, options={}}={}) {
     if ( !this.hasDamage ) throw new Error("You may not make a Damage Roll with this Item.");
 
+    // Fetch level from tags if not specified
+    let originalLevel = this.system.level;
+    let scaling = this.system.scaling;
+    const levelingFlag = this.getFlag("dnd5e", "spellLevel");
+    if ( !spellLevel && levelingFlag ) {
+      spellLevel = parseInt(levelingFlag.value);
+      originalLevel = parseInt(levelingFlag.base);
```

Comment:
Do we need to be lenient with these flag types? Is there any reason they wouldn't necessarily be the correct type already?

```suggestion
      spellLevel = levelingFlag.value;
      originalLevel = levelingFlag.base;
```